src/app/(app)/discover/person/[id]/page.tsx

```
@@ -0,0 +1,415 @@
+        const data = await res.json().catch(() => null);
+        throw new Error(data?.error || 'Failed to load person');
+      }
+      const data = await res.json();
+      if (signal.aborted) return;
+      setPerson(data);
+    } catch (err) {
+      if (err instanceof Error && err.name === 'AbortError') return;
+      if (signal.aborted) return;
+      setError(err instanceof Error ? err.message : 'Failed to load person');
+    } finally {
+      if (signal.aborted) return;
```

Biome (2.4.6)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)